perun-core/src/main/java/cz/metacentrum/perun/core/entry/GroupsManagerEntry.java

```diff
@@ -200,7 +200,7 @@ public void deleteGroups(PerunSession perunSession, List<Group> groups, boolean
 	}
 
 	@Override
-	public Group updateGroup(PerunSession sess, Group group) throws GroupNotExistsException, InternalErrorException, PrivilegeException {
+	public Group updateGroup(PerunSession sess, Group group) throws GroupNotExistsException, GroupExistsException, InternalErrorException, PrivilegeException {
```

Comment:
We can check this using SQL constraint itself in impl/dao layer, since we update name and description separately. We can catch ``DataIntegrityViolationException`` when updating name and then throw ``GroupExistsException`` rather than current ``InternalErrorException``. See example in ``FacilitiesManagerImpl`` method ``updateFacility()``.

Reply:
Yes, you are right. I changed that logic to be same as for Facility.